tests/Unit/GeneralAssemblyTest.php

```diff
@@ -166,4 +167,25 @@ public function test_passive_users_get_excused_automatically(): void
         $this->assertTrue($excused->contains($userPassive1) && $excused->contains($userPassive2));
         $this->assertNotNull($excused->first()->pivot->comment); // Check if excuse reason is set
     }
+
+    /**
+     * @return void
+     */
+    public function test_new_students_pass_requirements(): void
+    {
+        $user = User::factory()->create(['verified' => true]);
+        $user->educationalInformation()->save(EducationalInformation::factory()->make());
+        $user->educationalInformation->update(['year_of_acceptance' => now()->year]);
+
+        $generalAssembly = GeneralAssembly::factory()->create();
+        $generalAssembly2 = GeneralAssembly::factory()->create();
+
+        $generalAssembly->update(['closed_at' => now()->setYear(now()->year)->setMonth(2)->setDay(15)]);
+        $generalAssembly2->update(['closed_at' => now()->setYear(now()->year)->setMonth(9)->setDay(15)]);
+
+        $generalAssembly->presenceChecks()->create();
+        $generalAssembly2->presenceChecks()->create();
+
+        $this->assertTrue(GeneralAssembly::requirementsPassed($user));
```

Comment:
Also, it would be nice if you could add a case where the user does _not_ pass the requirements; e.g. by adding one more assembly in September.

Reply:
Done